external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/internal/OffsetManager.java

```diff
@@ -163,8 +167,9 @@ public long commit(OffsetAndMetadata committedOffset) {
 
         LOG.trace("{}", this);
         
-        LOG.debug("Committed [{}] offsets in the range [{}-{}] for topic-partition [{}].",
-                numCommittedOffsets, preCommitCommittedOffsets + 1, this.committedOffset, tp);
+        LOG.debug("Committed [{}] offsets in the range [{}-{}] for topic-partition [{}]."
+            + " Processing will resume at [{}] if the spout restarts.",
```

Comment:
at offset [{}] in case of a spout restart.

Reply:
Good catch, will fix